IdentityCore/src/MSIDOAuth2Constants.m

```diff
@@ -84,6 +84,7 @@
 NSString *const MSID_OAUTH2_ENVIRONMENT                  = @"environment";
 
 NSString *const MSID_PROTECTION_POLICY_REQUIRED          = @"protection_policy_required";
+NSString *const MSID_USER_DISPLAYABLE_IDENTIFIER         = @"adi";
```

Comment:
Just wondering, when do we need it?

Reply:
For True MAM CA scenarios, server returns this parameter so that we can return it to the application.